nautobot_device_lifecycle_mgmt/jobs/model_migration.py

```diff
@@ -1181,19 +1215,38 @@ def migrate_content_type_references_to_new_model(self, old_ct, new_ct, hide_chan
 
     def _create_placeholder_software_images(self):
         """Create placeholder software image files for software that is used by devices but no image currently exists."""
+        if version.parse(settings.VERSION) >= version.parse("2.3.1"):
+            self.logger.info("Skipping placeholder SoftwareImageFile creation. Nautobot version is 2.3.1 or later.")
```

Comment:
Given there is a similar message above, about Skipping should this match the same log level (debug)?